mm2src/common/common.rs

```diff
@@ -773,6 +771,7 @@ static mut PROCESS_LOG_TAIL: [u8; 0x10000] = [0; 0x10000];
 static TAIL_CUR: AtomicUsize = AtomicUsize::new(0);
 
 /// Keep a tail of the log in RAM for the integration tests.
+#[allow(static_mut_refs)] // TODO: Refactor PROCESS_LOG_TAIL to use lazystatic
```

Comment:
I think we need an issue for this and all the `static_mut_refs` uses to not forget about it

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/issues/2520